src/Microsoft.AspNetCore.SignalR.Core/DynamicHubClients.cs

```diff
@@ -15,17 +15,83 @@ public DynamicHubClients(IHubCallerClients clients)
             _clients = clients;
         }
 
+        /// <summary>
+        /// Gets a caller to all hub clients.
+        /// </summary>
+        /// <returns>A client caller.</returns>
         public dynamic All => new DynamicClientProxy(_clients.All);
+
+        /// <summary>
+        /// Gets a caller to all hub clients excluding the specified client connections.
```

Comment:
~~client~~ connections

(in all the below methods/properties/etc). A connection is always to a client, no need to repeat it :)